<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>361</width>
     <height>135</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">background-color : rgb(255, 255, 255);
font: 16pt &quot;나눔고딕 ExtraBold&quot;;</string>
      </property>
      <property name="text">
       <string>모든 암호는 암호화 되어 저장됩니다.</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">background-color : rgb(255, 255, 255);
font: 16pt &quot;나눔고딕 ExtraBold&quot;;</string>
        </property>
        <property name="text">
         <string>PW :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_ID"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_PW"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">background-color : rgb(255, 255, 255);
font: 16pt &quot;나눔고딕 ExtraBold&quot;;</string>
        </property>
        <property name="text">
         <string>ID :</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_saveLoginInfo">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : rgb(85, 85, 85);
color: rgb(255, 255, 255);
font: 9pt &quot;나눔고딕 ExtraBold&quot;;</string>
   </property>
   <property name="text">
    <string>저장(Enter)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
